### PR TITLE
add patient sample model and viewsets

### PR DESCRIPTION
**Pull request by anroopak**
Related Issue: https://github.com/coronasafe/care/issues/137

### CODE REVIEW

**Comment by bodhish** [comment]
@anroopak can you check  #151 also

**Comment by afzalIbnSH** [comment]
Is it better to map a sample to a consultation ? @anroopak

File: care/facility/models/patient_data.py [hunk 1]
```diff
@@ -66,6 +70,14 @@ class PatientTeleConsultation(models.Model):
     created_by = models.ForeignKey(User, on_delete=models.SET_NULL, null=True)
 
 
+class PatientSample(FacilityBaseModel):
+    patient = models.ForeignKey(PatientRegistration, on_delete=models.PROTECT)
+    facility = models.ForeignKey("Facility", on_delete=models.PROTECT)
+    result = models.IntegerField(choices=SAMPLE_TEST_RESULT_CHOICES, default=SAMPLE_TEST_RESULT_MAP["Awaiting"])
```
Comment (afzalIbnSH) [inline care/facility/models/patient_data.py]: Should we map this to `PatientConsultation` too ?

Comment (anroopak) [reply care/facility/models/patient_data.py]: @bodhish @tomahawk-pilot  What are your thoughts? I am thinking of adding.

**Comment by anroopak** [comment]
Updated PR: https://github.com/coronasafe/care/pull/160

**Pull request closed without merging**